### PR TITLE
modified: homePage mousemove && community module delete repeat message

### PR DESCRIPTION
**Pull request by jujuAnna**
1、 modify the top title "document" and "asf" when the mouse leave the title I want to close the list.
2、Deleted duplicate content：News
image
image

### CODE REVIEW

**Comment by jujuAnna** [comment]
nice work! I will review。

File: site_config/community.jsx [hunk 1]
```diff
@@ -4,15 +4,15 @@ export default {
...
-      },
+      // {
+      //   title: 'Community News',
+      //   children: [
+      //     {
+      //       title: 'Community News',
+      //       link: '/en-us/community/index.html',
+      //     },
+      //   ],
+      // },
```
Comment (jujuAnna) [inline site_config/community.jsx:15]: please delete

**Comment by GaoTianDuo** [comment]
nice work! I will review。

File: site_config/community.jsx (hunk 1 above)
Comment (GaoTianDuo) [inline site_config/community.jsx:15]: please delete

File: site_config/community.jsx [hunk 2]
```diff
@@ -210,15 +214,15 @@ export default {
...
-        ],
-      },
+      // {
+      //   title: '社区动态',
+      //   children: [
+      //     {
+      //       title: '社区动态',
+      //       link: '/zh-cn/community/team.html',
+      //     },
+      //   ],
```
Comment (GaoTianDuo) [inline site_config/community.jsx:224]: please delete

File: src/pages/docs/index.md.jsx [hunk 3]
```diff
@@ -26,7 +26,7 @@ const docsSource = {
   '1.3.4': docs134Config,
   '1.3.5': docs135Config,
   '1.3.6': docs136Config,
-  'dev': docsDevConfig,
+  dev: docsDevConfig,
```
Comment (GaoTianDuo) [inline src/pages/docs/index.md.jsx:29]: please rollback

**Review by GaoTianDuo — COMMENTED** [review]
please modify

**Comment by CalvinKirs** [comment]
PTAL @GaoTianDuo

**Review by GaoTianDuo — COMMENTED** [review]
nice work!

**Comment by GaoTianDuo** [comment]
@CalvinKirs  please review and merge

File: site_config/community.jsx [hunk 4]
```diff
@@ -4,15 +4,6 @@ export default {
...
     sidemenu: [
-      {
-        title: 'Community News',
-        children: [
-          {
-            title: 'Community News',
-            link: '/en-us/community/index.html',
-          },
-        ],
-      },
```
Comment (CalvinKirs) [inline site_config/community.jsx:15]: why do we need to delete this?

Comment (jujuAnna) [reply site_config/community.jsx:15]: Because the content of this module is duplicated with the content of the homepage, it has been deleted.

**Pull request merged**